com.github.foldynl.QLog.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.github.foldynl.QLog
```

Comment:
app-id MUST start with `io.github` as per the documentation

Reply:
```
{
    "errors": [
        "appid-uses-code-hosting-domain"
    ]
}
```

Reply:
I was not aware of this. If I remember correctly, this did not use to be the rule and I already have one Flatpak that uses the com.github.* namespace. Also, this project does not have any website (just the Git repo), so io.github.* does not make much sense to me. Anyway, I will take a look at the documentation and will rename the Flatpak accordingly.

Reply:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Thanks!

Reply:
it wasn't strictly enforced and since renaming is complicated we have added the existing one to the exception list. But no new package will be allowed to bypass the rule.

Reply:
Done.